datadog_checks_dev/datadog_checks/dev/tooling/commands/env/test.py

```diff
@@ -17,7 +17,6 @@
 @click.option(
     '--agent',
     '-a',
-    default='6',
```

Comment:
why?

Reply:
See commit message, @ChristineTChen can you make a separate PR?